docs/commerce/payment/create_payment_methods.md

```diff
@@ -8,9 +8,12 @@ edition: commerce
 If your [user role](work_with_permissions.md) includes the `Payment method/Create` permission, you can create payment methods. 
 With the `Payment method/Edit` permission, you can modify existing ones.
 
+Payment method definitions describe the way customers of an e-commerce presence pay for their orders during the checkout process.
```

Comment:
```suggestion
Payment methods describe the way shop customers pay for their orders during the checkout process.
```
Is there any business reason to not plainly say "shop"? "e-commerce presence" is too fancy by far, and really meaningful